mirdata/core.py

```diff
@@ -76,13 +84,15 @@ class Dataset(object):
 
     Attributes:
         data_home (str): path where mirdata will look for the dataset
+        index (dict or None): the dataset's file index
```

Comment:
index shouldn't be in attributes, since it's not an attriute! It's a parameter and it's documented there already